widgets/filters/src/FilterModal.tsx

```diff
@@ -29,13 +30,22 @@ const FilterModal: FunctionComponent<FilterModalProps> = ({
         initialFilter ? [...initialFilter.attrs_filter_rules, ...initialFilter.labels_filter_rules] : []
     );
     const { errors, setErrors, clearErrors, setMessageAsError } = useErrors();
+    const [filterRulesInvalid, setFilterRulesInvalid] = useState<boolean>(false);
```

Comment:
Optional:

The generic parameter can be inferred from the type of the default value :slightly_smiling_face: 

```suggestion
    const [filterRulesInvalid, setFilterRulesInvalid] = useState(false);
```

Reply:
Right, good point, removed `<boolean>`.